src/app.js

```diff
@@ -0,0 +1,124 @@
+import dots from './dots.svg';
```

Comment:
- [ ] Great job completing your app, it's looking great! However, the localStorage is not being updated when you edit a task or when you check the checkbox:

Edit task not updated in localStorage:

![Captura de pantalla 2022-06-03 a la(s) 9 58 03 p  m](https://user-images.githubusercontent.com/83717285/171977239-bbf1224a-7755-4c33-8ef4-24e8e02f5bc3.png)

Checked checkbox not updating completed status:

![Captura de pantalla 2022-06-03 a la(s) 9 56 36 p  m](https://user-images.githubusercontent.com/83717285/171977412-a619fe0b-5a9a-4dea-a479-4c972c438888.png)

- [ ] It would be great to fix the edit functionality, the checkbox completed status update problem the clear all completed functionality which is not working because the completed status is not being updated.

Reply:
Could you kindly check the task for this milestone? Updating status is considered for the next milestone. And as for local storage, changing indexes is required only. I would appreciate it if I'm allowed to complete new functionality accordingly to the corresponding milestone.

Reply:
Hi @petushka1,

you are right, I made a mistake, that is not supposed to be a requirement for this milestone. 
I apologize for this, can you please submit the PR for another review? The next code reviewer will fix this misunderstanding.

Reply:
You don't need to apologise. Things happen, you have multiple reviews from different milestones. Anyway, your input about 'edit' function makes sense thus I just pushed these changes. Thanks